### PR TITLE
Updates to Ranking Criteria

### PR DESCRIPTION
**Pull request by pishifat**
cut song guideline https://osu.ppy.sh/community/forums/topics/965954
difficulty name guideline https://osu.ppy.sh/community/forums/topics/968278
drain/play time rule https://osu.ppy.sh/community/forums/topics/996501
metadata markers https://osu.ppy.sh/community/forums/topics/942626
backgroudn requirements https://osu.ppy.sh/community/forums/topics/997606

### CODE REVIEW

File: wiki/Ranking_Criteria/en.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ All forms of Artist/Title standardisation apply to both the `Romanised` and `Uni
 - **When a fictional character is credited as the singer of a song, the artist field is to be formatted in a `Character (CV: Voice Actor)` format.** For live action, credit the voice actor only.
 - **If the song is `TV size`, use a `(TV Size)` marker at the end of the current title string.** If there is an existing `TV size` marker in the title, the `(TV Size)` marker would replace it. Additionally, songs with existing `Short Ver` or `Game Ver` markers in their titles must use `(Short Ver.)` or `(Game Ver.)` markers.
   - Note: If a mapset's song contains matching sections to the `TV size`/`Short Ver`/`Game Ver` song, in the same order, and is roughly the same length as the official cut song, the mapset's edit will also count as a `TV Size`/`Short Ver`/`Game Ver` respectively. Covers and remixes do not count.
+- **If a song has been edited to have a higher tempo, use a `(Sped Up Ver.)` marker at the end of the current title string.** If there is an existing `Sped Up Ver` marker in the title, the `(Speed Up Ver.)` marker would replace it. Sped up songs in Techno, Trance, Hardstyle, or other similar genres must use a `(Nightcore Mix)` marker instead.
```
Comment (cl8n) [inline wiki/Ranking_Criteria/en.md]: last part (or just the "must" rly) doesnt seem decided in the thread

Comment (cl8n) [reply wiki/Ranking_Criteria/en.md]: also u misspelled the third instance of tag

File: wiki/Ranking_Criteria/en.md [hunk 2]
```diff
@@ -228,18 +229,27 @@ This category contains explicit allowance statements of concepts and rules that
...
 - **Cuts should not be shorter than 1:30.** These often cause unsatisfying play experiences and are generally seen as disrespectful to the artists. This does not apply to official cuts or recreations of official cuts.
-- **Cuts should maintain the general impression and intensity of the full song.** Cuts that make significant changes to the overall listening experience of the song are very misleading and often cause unsatisfying play experiences. This does not apply to official cuts or recreations of official cuts.
+- **Cuts should maintain the general impression and intensity of the full song.** Cuts that change the structure of the full song (such as excluding or rearranging a song's intro/verse/chorus/outro) can lead to mispresentation of it and often cause unsatisfying playing experiences. This does not apply to official cuts or recreations of official cuts.
 
 ## Video and background
 
 ### Rules
 
 - **You must have a background image on every difficulty of your beatmap.** Different background files for different difficulties is acceptable.
+- **Background image requirements:**
```
Comment (cl8n) [inline wiki/Ranking_Criteria/en.md]: make it a sentence

File: wiki/Ranking_Criteria/en.md [hunk 3]
```diff
@@ -53,18 +53,18 @@ For the record, these include the recent changes made [from the discussion forum
...
 - **A beatmapset host must have beatmapped equal or more difficulties than any guest difficulty beatmap creator.** This is to provide credit where credit is due. Collab difficulties are only considered partial difficulties, and drain time will be used to determine the amount of contribution when a guest has beatmapped significantly more than the host.
 - **A beatmapset host and guest beatmap creators can make changes to their respective difficulties as they wish.** If there is a disagreement between both, the beatmapset host must delete the guest contribution upon request. If a guest beatmap creator cannot be contacted for a month, they will be assumed to agree with any changes.
 
 ### Guidelines
 
+- **A beatmapset's custom difficulty naming must follow a common theme or pattern related to the song or difficulty and must not be misrepresentative.** A difficulty name is misrepresentative if it implies a different difficulty level (e.g. naming an "Expert" difficulty as "Normal").
```
[the remark this answers is not in the document]
Comment (pishifat) [reply wiki/Ranking_Criteria/en.md]: changing to rule then

File: wiki/Ranking_Criteria/en.md (hunk 1 above)
Comment (pishifat) [reply wiki/Ranking_Criteria/en.md]: brought it up on thread

**Comment by cl8n** [comment]
is this done now

**Comment by pishifat** [comment]
yes

**Pull request merged**